prime-router/src/test/kotlin/fhirengine/utils/FhirTranscoderTests.kt

```diff
@@ -1,13 +1,73 @@
 package gov.cdc.prime.router.fhirengine.utils
 
 import assertk.assertThat
+import assertk.assertions.contains
+import assertk.assertions.isNotEmpty
 import assertk.assertions.isNotNull
+import org.hl7.fhir.r4.model.Bundle
+import org.json.JSONException
+import org.json.JSONObject
+import java.util.Date
 import kotlin.test.Test
+import kotlin.test.assertTrue
 
 class FhirTranscoderTests {
     @Test
     fun `test get message engine`() {
         // Just make sure we can get an engine, which means the config is setup correctly.
         assertThat(FhirTranscoder.getMessageEngine()).isNotNull()
     }
+
+    @Test
+    fun `test default parser encode FHIR bundle`() {
+        fun isValidJSONObject(jsonVal: String): Boolean {
+            try {
+                JSONObject(jsonVal)
+            } catch (ex: JSONException) {
+                return false
+            }
+            return true
+        }
+        // Create test FHIR Bundle
+        val testBundle = Bundle()
+        testBundle.type = Bundle.BundleType.MESSAGE
+        testBundle.id = "someid"
+        testBundle.timestamp = Date()
+        testBundle.addEntry()
+
+        val encodedBundle = FhirTranscoder.encode(testBundle)
+
+        assertThat(encodedBundle).isNotEmpty()
+        assertTrue { isValidJSONObject(encodedBundle) }
+
+        // Ensure contains id and type
+        assertThat(encodedBundle).contains("\"id\":\"${testBundle.id}\"")
```

Comment:
Don't do string comparison in JSON.  Use some utility to grab the property you want then check the value.  Main reason here is that if someone turns on pretty printing of that json then the spacing in the JSON will change.